editor/filesystem_dock.cpp

```diff
@@ -1225,6 +1225,31 @@ void FileSystemDock::_try_duplicate_item(const FileOrFolder &p_item, const Strin
 			if (err != OK) {
 				EditorNode::get_singleton()->add_io_error(TTR("Error duplicating:") + "\n" + old_path + ".import\n");
 			}
+
+			// Remove uid from .import file to avoid conflict.
+			Ref<ConfigFile> cfg;
+			cfg.instantiate();
+			cfg->load(new_path + ".import");
+			cfg->erase_section_key("remap", "uid");
+			cfg->save(new_path + ".import");
+		} else if (p_item.is_file && (old_path.get_extension() == "tscn" || old_path.get_extension() == "tres")) {
+			// FIXME: Quick hack to fix text resources. This should be fixed properly.
+			Error err;
```

Comment:
```
editor/filesystem_dock.cpp: In member function 'void FileSystemDock::_try_duplicate_item(const FileSystemDock::FileOrFolder&, const String&) const':
editor/filesystem_dock.cpp:1237:10: error: declaration of 'err' shadows a previous local [-Werror=shadow=local]
 1237 |    Error err;
      |          ^~~
editor/filesystem_dock.cpp:1220:8: note: shadowed declaration is here
 1220 |  Error err = p_item.is_file ? da->copy(old_path, new_path) : da->copy_dir(old_path, new_path);
      |        ^~~
```
You can likely just reuse the existign one.

Reply:
Ah, I didn't notice that.